src/chunk/render.rs

```diff
@@ -45,57 +45,126 @@ macro_rules! build_chunk_pipeline {
                 ..PipelineDescriptor::new(ShaderStages {
                     vertex: shaders
                         .add(Shader::from_glsl(ShaderStage::Vertex, include_str!($file))),
-                    fragment: Some(shaders.add(Shader::from_glsl(
-                        ShaderStage::Fragment,
-                        include_str!("tilemap.frag"),
-                    ))),
+                    fragment: Some(shaders.add(Shader::from_glsl(ShaderStage::Fragment, {
+                        let _wasm = false;
+                        #[cfg(target_arch = "wasm32")]
+                        let _wasm = true;
+
+                        if _wasm {
+                            include_str!("render_web/tilemap.frag")
+                        } else {
+                            include_str!("render_native/tilemap.frag")
+                        }
```

Comment:
There is a better way to handle this with `if cfg!(target_arch = "wasm32") {`

Reply:
Yeah, this part wasn't very ideal. The compiler kept bullying me here with conditionals applied directly to a macro, but I probably just did something wrong.